contracts/contracts/interfaces/IVault.sol

```diff
@@ -210,4 +216,13 @@ interface IVault {
     function setAdminImpl(address) external;
 
     function removeAsset(address _asset) external;
+
+    function addStrategyToMintWhitelist(address strategyAddr) external;
+
+    function removeStrategyFromMintWhitelist(address strategyAddr) external;
+
+    function mintWhitelistedStrategy(address strategyAddr)
```

Comment:
maybe a `isMintWhitelistedStrategy` might be a bit clearer?

Reply:
Good point, Will rename it

Reply:
Have renamed it